packages/@lwc/engine-core/src/framework/def.ts

```diff
@@ -47,11 +48,6 @@ import {
 } from '../shared/circular-module-dependencies';
 import { getComponentOrSwappedComponent } from './hot-swaps';
 
-export enum RenderMode {
-    Light,
-    Shadow,
-}
-
```

Comment:
`RenderMode` was defined in two places, which works fine with `enum` but not `const enum` since the types don't match.